packages/core/redux/eos/constants.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+export const DUCK_EOS = 'eos'
+export const BLOCKCHAIN_EOS = 'EOS'
+export const EOS = 'EOS'
+
+export const EOS_UPDATE = 'EOS/UPDATE'
+export const EOS_UPDATE_TOKEN = 'EOS/TOKEN/UPDATE'
+export const EOS_UPDATE_WALLET = 'EOS/WALLET/CREATE'
+export const TX_CREATE = 'EOS/TX/CREATE'
+export const TX_REMOVE = 'EOS/TX/REMOVE'
+export const TX_UPDATE = 'EOS/TX/UPDATE'
+
+export const EOS_NETWORK_CONFIG = {
```

Comment:
It is better to keep all such configs in the `packages/login/network/settings.js`